tests/integration/031_vault_credentials/expected_outputs.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
why do we need that?

Reply:
This is required for all tests even if not used. Otherwise test fail.
It's used if you want to test with terraform's "output".